### PR TITLE
Matching landing page title to subheadings

### PR DESCRIPTION
**Pull request by alexlindroos**
Screenshot 2021-04-13 at 12 07 57

@kenzieschmoll I matched the title of the landing page to subheadings. If we want to make subheadings a little bit smaller, I can change them i.e to 'headline6'

### CODE REVIEW

File: packages/devtools_app/lib/src/scaffold.dart [hunk 1]
```diff
@@ -398,7 +398,7 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
       automaticallyImplyLeading: false,
       title: Text(
         title,
-        style: Theme.of(context).textTheme.subtitle1,
+        style: Theme.of(context).textTheme.headline5,
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/scaffold.dart:401]: Let's do headline6 here. Thanks!

**Review by kenzieschmoll — APPROVED** [review]

File: packages/devtools_app/lib/src/scaffold.dart (hunk 1 above)
Comment (alexlindroos) [reply packages/devtools_app/lib/src/scaffold.dart]: Okay, I updated it now.

**Comment by kenzieschmoll** [comment]
Looks like the bots are hitting a formatting issue with 

Changed lib/src/memory/memory_heap_tree_view.dart
Changed lib/src/memory/memory_screen.dart

if you are not merged to DevTools master, please try that, and then flutter format these two files.

**Comment by alexlindroos** [comment]
> Looks like the bots are hitting a formatting issue with
> 
> ```
> lib/src/memory/memory_heap_tree_view.dart
> lib/src/memory/memory_screen.dart
> ```
> 
> if you are not merged to DevTools master, please try that, and then flutter format these two files.

Done, still few bots failing.

**Pull request merged**